Files/UserControls/FileIcon.xaml.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Files.View_Models;
+using Windows.UI.Xaml.Controls;
+
+namespace Files.UserControls
+{
+    public sealed partial class FileIcon : UserControl
+    {
+        private double itemSize;
+
+        public SelectedItemsPropertiesViewModel ViewModel { get; set; }
+        public double ItemSize { get; set; }
+
+        public double Size
```

Comment:
Seems like this property is not used, so could you remove it?

Reply:
Of curse